benchmark/poseidon/poseidon_benchmark_config.h

```diff
@@ -0,0 +1,61 @@
+#ifndef BENCHMARK_POSEIDON_POSEIDON_BENCHMARK_CONFIG_H_
+#define BENCHMARK_POSEIDON_POSEIDON_BENCHMARK_CONFIG_H_
+
+#include <stddef.h>
+
+#include <string>
+
+#include "tachyon/base/console/iostream.h"
+#include "tachyon/base/flag/flag_parser.h"
+
+namespace tachyon {
+
+class PoseidonBenchmarkConfig {
+ public:
+  PoseidonBenchmarkConfig() = default;
+  PoseidonBenchmarkConfig(const PoseidonBenchmarkConfig& other) = delete;
+  PoseidonBenchmarkConfig& operator=(const PoseidonBenchmarkConfig& other) =
+      delete;
+
+  bool check_results() const { return check_results_; }
+  size_t repeating_num() const { return repeating_num_; }
+  size_t absorbing_num() const { return absorbing_num_; }
+  size_t squeezing_num() const { return squeezing_num_; }
+
+  bool Parse(int argc, char** argv) {
+    base::FlagParser parser;
+
+    parser.AddFlag<base::Flag<size_t>>(&repeating_num_)
+        .set_short_name("-n")
+        .set_help("Specify the number of repetition 'n'. By default, 10.");
+    parser.AddFlag<base::Flag<size_t>>(&absorbing_num_)
+        .set_short_name("-a")
+        .set_help("Specify the number of absorptions 'a'. By default, 100.");
+    parser.AddFlag<base::Flag<size_t>>(&squeezing_num_)
+        .set_short_name("-s")
+        .set_help("Specify the number of squeeze 's'. By default, 100.");
+    parser.AddFlag<base::Flag<bool>>(&check_results_)
+        .set_long_name("--check_results")
+        .set_help("Whether checks results generated by each poseidon runner.");
+
+    {
+      std::string error;
+      if (!parser.Parse(argc, argv, &error)) {
+        tachyon_cerr << error << std::endl;
+        return false;
+      }
+    }
+
+    return true;
```

Comment:
It's a minor, but for consistency, Could you move this to `poseidon_benchmark_config.cc`? And for msm, it's named `MSMConfig` and why did you name it `PosiedonBenchmarkConfig`?

Reply:
Yup, I'll do.

It's actually named by @dongchangYoo and I think it's because we have `crypto::PoseidonConfig` with the same name. Should I change it to `PoseidonConfig` for consistency and since they are in different namespace?

Reply:
Yeap, for me, that makes more sense